compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/window/ComposeWindow.js.kt

```diff
@@ -228,6 +234,16 @@ private class ComposeWindow(
             val processed = layer.view.onKeyboardEventWithResult(event.toSkikoEvent(SkikoKeyboardEventKind.UP))
             if (processed) event.preventDefault()
         }
+
+        window.addEventListener("focus", {
+            lifecycleOwner.handleLifecycleEvent(Lifecycle.Event.ON_RESUME)
+        })
+
+        window.addEventListener("blur", {
+            lifecycleOwner.handleLifecycleEvent(Lifecycle.Event.ON_PAUSE)
+        })
+
+        lifecycleOwner.handleLifecycleEvent(Lifecycle.Event.ON_CREATE)
```

Comment:
Let's not call `ON_CREATE` and `ON_START` in the same method `init`, and just call `ON_START` straight away in the end of `init`, if JS doesn't have meaning for `CREATED` state.